layouts/shortcodes/ct_logs.html

```diff
@@ -1,8 +1,29 @@
-{{ range $.Site.Data.transparency.transparencyLogs }}
-<p>
-  <strong>{{ i18n "certificate_transparency_name" }}{{ .Site.Params.beforeColon | safeHTML }}: {{ .name }}</strong><br>
-  {{ i18n "certificate_transparency_role" }}{{ .Site.Params.beforeColon | safeHTML }}: {{ if eq .role "production" }}{{ i18n "certificate_transparency_role_production" }}{{ end }}{{ if eq .role "test" }}{{ i18n "certificate_transparency_role_test" }}{{ end }}<br>
-  {{ i18n "certificate_transparency_uri" }}{{ .Site.Params.beforeColon | safeHTML }}: {{ .uri }}<br>
-  {{ i18n "certificate_transparency_public_key" }}{{ .Site.Params.beforeColon | safeHTML }}: <code>{{ .publicKey }}</code>
-</p>
+{{ range $.Site.Data.transparency.categories }}
+    {{ $category := . }}
+    {{ if eq $category "production" }}
+        <h3>{{ i18n "certificate_transparency_role_production" }}</h3>
+        <ul>
+            <li>Our production ACME API environment submits certificates here</li>
+            <li>Will eventually be incorporated in browser CT programs</li>
+        </ul>
+    {{ else if eq $category "testing" }}
+        <h3>{{ i18n "certificate_transparency_role_test" }}</h3>
+        <ul>
+            <li>We test new versions of <a href="http://github.com/google/trillian">Trillian</a> and <a href="https://github.com/google/certificate-transparency-go">certificate-transparency-go</a> here before deploying them to production</li>
+            <li>Both the production and staging ACME API environments submit certificates to Testflume</li>
+            <li>Testflume's accepted roots list includes all of the Oak accepted roots</li>
+            <li>Can be used by other certificate authorities for testing purposes</li>
+        </ul>
```

Comment:
I think the `<ul>` section should be inside the `content/en/docs/ct-logs.html` page to allow translation.

This can be replaced by `{{ .Inner }}` and in the `ct-logs.html` : 

```hugo
{{< ct_logs >}}
<ul>
...
</ul>
{{< /ct_logs >}}.
```

NB: you may have to update translation to add `{{< /ct_logs >}}` (even empty) to avoid build fail.

Reply:
(if you don't feel comfortable doing it, I can do it after the merge)

Reply:
Thanks! I was curious how to go about doing this. I'll give it a shot.

Reply:
You have an example in https://github.com/letsencrypt/website/blob/master/content/en/docs/client-options.md#acmev1-and-acmev2 🙂

Reply:
That was a pretty gnarly rewrite to use `.Inner`. I think it took like 4 hours, 2 beers, and my wife's web developer eyes also staring at it.